manifests/s/StaticalApS/PrismforALStandardEdition(SE)/2.2.0.6/StaticalApS.PrismforALStandardEdition(SE).locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: StaticalApS.PrismforALStandardEdition(SE)
+PackageVersion: 2.2.0.6
+PackageLocale: en-US
+Publisher: Statical ApS
+PackageName: Prism for AL Standard Edition (SE)
+License: https://stati-cal.com/Statical-Prism-EULA-3.1.pdf
+ShortDescription: Statical Prism for AL Standard Edition (SE)

```

Comment:
Please provide a better short description of what Statical Prism for AL does or is used for